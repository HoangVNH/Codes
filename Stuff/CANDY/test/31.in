10
47
VVVVVVVVXXXDDDDDDDDDDVVVXDDDDDXXXXXVVVVVDDXXXXX
39
XXVVVXXXXVXXXXXDXXXXXXXXXXXDDDDVVVVDDDD
34
DDVXXXXXXXVVVVVVVVVVVVDDDDDXXXXXDV
24
DDDDDDDVVXXXXXXXDDVVVDDD
23
XVVVVVVVDDXXXXXXXXXXXXX
25
XXXXXXXDXVVVVVDDVVVVVVVVV
43
VVVVXXXXDDDXXVVVVVVVVVVVXXXDDDDDDDVDDDDDDDD
44
DDDVVVVDDDVVXXXXDVVVVVVXDDDVVVXXXXXDDDDXXXXX
21
DVVVVVVDDDDDDVVXXXXDD
35
VVVVVVVDDDDVVVDDDDDVVVVVDXXXXDDDDDV
