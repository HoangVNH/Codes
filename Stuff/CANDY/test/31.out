46
38
32
23
22
24
42
43
20
34
